libs/application/templates/european-health-insurance-card/src/lib/dataSchema.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod'
+
+export const contactSchema = z.object({
+  name: z.string().min(1),
+})
+
+export const dataSchema = z.object({
```

Comment:
Is there no more validation for the whole application?
I think most of your checkbox are optional at this point meaning the applicant could pass through your application not answering them at all.